### PR TITLE
fix words misuse

### PR DESCRIPTION
**Pull request by ZaraLang**
**What does this PR do?**
fix words misuse
**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/src/main/java/org/tron/common/application/ApplicationImpl.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public void shutdown() {
     dbManager.stopRePushThread();
     dbManager.stopRePushTriggerThread();
     EventPluginLoader.getInstance().stopPlugin();
-    logger.info("******** end to shutdown ********");
+    logger.info("******** stop to shutdown ********");
```
Comment (andelf) [inline framework/src/main/java/org/tron/common/application/ApplicationImpl.java]: It seems that this got the wrong meaning.

"stop to shutdown" = "stop something (which is happening now) and then shutdown"

File: framework/src/main/java/org/tron/common/backup/BackupManager.java [hunk 2]
```diff
@@ -72,7 +72,7 @@ public void init() {
     try {
       localIp = InetAddress.getLocalHost().getHostAddress();
     } catch (Exception e) {
-      logger.warn("Get local ip failed.");
+      logger.warn("Getting local ip failed.");
```
Comment (andelf) [inline framework/src/main/java/org/tron/common/backup/BackupManager.java]: "Failed to get local IP"

File: framework/src/main/java/org/tron/common/overlay/server/PeerConnectionCheckService.java [hunk 3]
```diff
@@ -41,7 +41,7 @@
 
   @PostConstruct
   public void check() {
-    logger.info("start the PeerConnectionCheckService");
+    logger.info("start the PeerConnectionCheck service");
```
Comment (andelf) [inline framework/src/main/java/org/tron/common/overlay/server/PeerConnectionCheckService.java]: Actually there's a `PeerConnectionCheckService` class. so the original is OK.

File: framework/src/main/java/org/tron/common/application/ApplicationImpl.java (hunk 1 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/common/application/ApplicationImpl.java]: ok, got it, will change

File: framework/src/main/java/org/tron/common/backup/BackupManager.java (hunk 2 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/common/backup/BackupManager.java]: perfect!

File: framework/src/main/java/org/tron/common/overlay/server/PeerConnectionCheckService.java (hunk 3 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/common/overlay/server/PeerConnectionCheckService.java]: ok

File: framework/src/main/java/org/tron/core/Wallet.java [hunk 4]
```diff
@@ -1379,11 +1379,11 @@ private void validateInput(OutputPointInfo request) throws BadItemException, Zks
       throw new ZksnarkException(SHIELDED_ID_NOT_ALLOWED);
     }
     if (request.getBlockNum() < 0 || request.getBlockNum() > 1000) {
-      throw new BadItemException("request.BlockNum must be specified with range in【0，1000】");
+      throw new BadItemException("request.BlockNum must be specified with range in [0, 1000]");
     }
 
     if (request.getOutPointsCount() < 1 || request.getOutPointsCount() > 10) {
-      throw new BadItemException("request.OutPointsCount must be speccified with range in【1，10】");
+      throw new BadItemException("request.OutPointsCount must be speccified with range in[1, 10]");
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/Wallet.java]: space before `[`

File: framework/src/main/java/org/tron/core/db/Manager.java [hunk 5]
```diff
@@ -791,10 +791,10 @@ public synchronized void eraseBlock() {
     try {
       BlockCapsule oldHeadBlock = getBlockById(
           getDynamicPropertiesStore().getLatestBlockHeaderHash());
-      logger.info("begin to erase block:" + oldHeadBlock);
+      logger.info("start to erase block:" + oldHeadBlock);
       khaosDb.pop();
       revokingStore.fastPop();
-      logger.info("end to erase block:" + oldHeadBlock);
+      logger.info("stop to erase block:" + oldHeadBlock);
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/db/Manager.java]: ditto

File: framework/src/main/java/org/tron/core/db/Manager.java [hunk 6]
```diff
@@ -852,8 +852,8 @@ private void switchFork(BlockCapsule newHead)
               newHead.getBlockId(), getDynamicPropertiesStore().getLatestBlockHeaderHash());
     } catch (NonCommonBlockException e) {
       logger.info(
-          "this is not the most recent common ancestor, need to remove all "
-              + "blocks in the fork chain.");
+          "this is not the most recent common ancestor, "
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/db/Manager.java]: useless style change.

File: framework/src/main/java/org/tron/core/db/Manager.java [hunk 7]
```diff
@@ -1696,7 +1696,7 @@ private void reorgContractTrigger() {
           postContractTrigger(trx.getTrxTrace(), true);
         }
       } catch (BadItemException | ItemNotFoundException e) {
-        logger.error("block header hash not exists or bad: {}",
+        logger.error("block header hash does not exist or bad: {}",
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/db/Manager.java]: block header hash does not exist or bad
> block header hash does not exist or is bad

File: framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java [hunk 8]
```diff
@@ -104,7 +104,7 @@ public void doBackup(BlockCapsule block) {
     logger
         .info("current block number is {}, backup all store use {} ms!", block.getNum(), timeUsed);
     if (timeUsed >= 3000) {
-      logger.warn("backup db use too much time.");
+      logger.warn("backup db uses too much time.");
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java]: s/backup/backing up/

**Comment by andelf** [comment]
Thanks for your effort. 😃

File: framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java (hunk 8 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java]: done

File: framework/src/main/java/org/tron/core/db/Manager.java (hunk 6 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/core/db/Manager.java]: just want to start a new line after the comma, looks friendly

File: framework/src/main/java/org/tron/core/db/Manager.java (hunk 7 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/core/db/Manager.java]: done

File: framework/src/main/java/org/tron/core/Wallet.java (hunk 4 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/core/Wallet.java]: done

**Comment by ZaraLang** [comment]
I don't know why the first one "buildkite/interface-status-test-in-docker" always failed…

**Comment by MiraculousWang** [comment]
> I don't know why the first one "buildkite/interface-status-test-in-docker" always failed…

sorry about that, We have solved the problem

File: framework/src/main/java/org/tron/core/db/Manager.java (hunk 5 above)
Comment (ZaraLang) [reply framework/src/main/java/org/tron/core/db/Manager.java]: done

**Pull request merged**